scripts/test/__init__.py

```diff
@@ -76,9 +61,6 @@ def test_all(clean_cache: bool = False):
 
     steps.extend(
         [
-            _step.create_mypy_pkg_file,
-            _step.mypy_src,
```

Comment:
can you keep this step in there?

Reply:
Thank you, I removed that line accidentally.